assets/src/data/model/attendee/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * External imports
+ */
+import { isUndefined } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+	getQueryString as baseGetQueryString,
+	QUERY_ORDER_ASC,
+	ALLOWED_ORDER_VALUES,
+} from '../base';
+
+// import from constants file when that gets added
+const REGISTRATION_STATUS_IDS = [
+	'RAP',
+	'RCN',
+	'RDC',
+	'RIC',
+	'RNA',
+	'RPP',
+	'RWL',
+];
```

Comment:
I think it's be good if this was part of the registration model and then pulled in here from there.  I imagine other models might need this as well.  So what you could do is get the model pull you did (#553 - its approved) merged into GB/master and then into this pull and then move this constant into the model/registration/constants.js file?

Reply:
I know in the other pull the _individual status constants_ are in the status model and that's okay and I think it should stay that way.  However the specific code I'm referring to is this:
```
const REGISTRATION_STATUS_IDS = [
+	'RAP',
+	'RCN',
+	'RDC',
+	'RIC',
+	'RNA',
+	'RPP',
+	'RWL',
+];
```

This is different in that its used to provide an _array_ of registration statuses.   Since its informative about the `registration` model domain, I thought it would be useful if its actually kept with the registration model.  I can totally see other models like Transaction, or Ticket that might build queries relying on a `registration_status` prop in which case it would make sense to pull the specific allowed props enum from the registration model.

So to be clear, this array is acting more as an `enum` definition and thus I think it should reside in the model it relates to.

With that said, if you want to put all the specific status enum-link arrays like this in the status model I'm also fine with that, you could label the constants appropriately in it.